packages/website/pages/components/[slug].tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import ErrorPage from 'next/error';
+import { remark } from 'remark';
+import html from 'remark-html';
+import { css } from 'emotion';
+
+import * as f36Components from '@contentful/f36-components';
+import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
+
+import { geltAllMDX, getPageBySlug } from '../../utils/content';
+
+const styles = {
+  root: css({
+    minHeight: '100vh',
```

Comment:
Why do we need this?

Reply:
It's just an example of emotion. It will be removed when we style the layout properly.